bears/python/PEP8NotebookBear.py

```diff
@@ -0,0 +1,130 @@
+import autopep8
+from copy import copy
+import nbformat
+
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class NotebookHandler:
+    """Utility class for Jupyter Notebook handling."""
+
+    def __init__(self, filename, file):
+        """
+        Constructs a notebook handler to deal with NotebookNode objects.
+        A NotebookNode is a dict-like node with attribute-access.
+
+        :param filename: The filename of the file.
+        :param file:     The file contents as string array.
+        """
+        self.filename = filename
+        self.file = file
+        self._file_nb = self.read_from_list(file)
+
+    @staticmethod
+    def read_from_list(file):
```

Comment:
since nobody else uses it probably not ;)
I will change it to `_read_from_list`.